freeipa/src/main/java/com/sequenceiq/freeipa/flow/instance/reboot/RebootEvent.java

```diff
@@ -3,11 +3,15 @@
 import com.sequenceiq.cloudbreak.cloud.event.CloudPlatformResult;
 import com.sequenceiq.cloudbreak.cloud.event.instance.RebootInstancesResult;
 import com.sequenceiq.flow.core.FlowEvent;
+import com.sequenceiq.freeipa.flow.stack.HealthCheckFailed;
+import com.sequenceiq.freeipa.flow.stack.HealthCheckSuccess;
 
 public enum RebootEvent implements FlowEvent {
     REBOOT_EVENT("REBOOT_TRIGGER_EVENT"),
     REBOOT_FINISHED_EVENT(CloudPlatformResult.selector(RebootInstancesResult.class)),
     REBOOT_FAILURE_EVENT(CloudPlatformResult.failureSelector(RebootInstancesResult.class)),
+    REBOOT_WAIT_UNTIL_AVAILABLE_FINISHED_EVENT(CloudPlatformResult.selector(HealthCheckSuccess.class)),
```

Comment:
I would prefer using `EventSelectorUtil` to `CloudPlatformResult` in this class. I know the result is the same.